components/nimbus/tests/common/mod.rs

```diff
@@ -2,43 +2,99 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+#[cfg(feature = "rkv-safe-mode")]
+use rkv::StoreOptions;
+
 // utilities shared between tests
-use nimbus::{error::Result, AppContext, NimbusClient, RemoteSettingsConfig};
+cfg_if::cfg_if! {
+    if #[cfg(feature = "stateful")] {
```

Comment:
Where the whole file is behind this, would it make more sense to change the `use` site(s)— in the same way `nimbus_client.rs` doesn't have any cfg/feature attributes?